com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -1234,6 +1255,18 @@ static void UpgradeAsset(int assetInstanceID)
                 asset.k_AssetPreviousVersion = 10;
             }
 
+            if(asset.k_AssetPreviousVersion < 11)
+            {
```

Comment:
Is this a correct way to Reload PostProcessingData asset that now has to have links to blue noise and bayer matrix texture?

Reply:
> Is this a correct way to Reload PostProcessingData asset that now has to have links to blue noise and bayer matrix texture?

Moved textures to the asset itself as post processing data might not be assigned at all.